### PR TITLE
Add governance

### PR DESCRIPTION
**Pull request by knlambert**
## Description

See https://github.com/cncf/toc/issues/1589.

Waiting after @cindymullins-dw to create the missing slack channels. Feel free to comment to improve this.

### CODE REVIEW

**Review by njayp — APPROVED** [review]

File: MAINTAINERS.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Maintainers are listed in alphabetical order.
 * Kévin Lambert, [knlambert](https://github.com/knlambert), <klambert@datawire.io>
 * Nick Powell, [njayp](https://github.com/njayp), <nickpowell@datawire.io>
 * Thomas Hallgren, [thallgren](https://github.com/thallgren), <thomas@datawire.io>
+* Guillaume Veschambre, [shepz](https://github.com/shepz), <gveschambre@datawire.io>
+* Sarabraj Singh, [sarabrajsingh](https://github.com/sarabrajsingh), <rsingh@datawire.io>
```
Comment (thallgren) [inline MAINTAINERS.md:15]: The heading states "alphabetical order", so this list must be reordered.

Comment (thallgren) [inline MAINTAINERS.md]: The heading states "alphabetical order", so this list must be reordered.
```suggestion
* Thomas Hallgren, [thallgren](https://github.com/thallgren), <thomas@datawire.io>
* Kévin Lambert, [knlambert](https://github.com/knlambert), <klambert@datawire.io>
* Nick Powell, [njayp](https://github.com/njayp), <nickpowell@datawire.io>
* Sarabraj Singh, [sarabrajsingh](https://github.com/sarabrajsingh), <rsingh@datawire.io>
* Guillaume Veschambre, [shepz](https://github.com/shepz), <gveschambre@datawire.io>
```

**Review by thallgren — CHANGES_REQUESTED** [review]
Please also remove the old GOVERNANCE.md

**Review by sarabrajsingh — APPROVED** [review]

**Review by thallgren — APPROVED** [review]

**Comment by thallgren** [comment]
Consider using last name for alphabetic order (as per my proposal). I think it's far more common. And perhaps order the emiriti list the same way.

**Review by josecv — APPROVED** [review]

**Review by shepz — APPROVED** [review]

**Pull request merged**